lib/hydra/works/services/generic_file/persist_derivatives_output_file.rb

```diff
@@ -0,0 +1,22 @@
+require 'hydra/derivatives'
+
+module Hydra::Works
+  class PersistDerivativesOutputFile < Hydra::Derivatives::PersistOutputFileService
+
+    # Persists a Derivative to a GenericFile
+    # @param [Hydra::Works::GenericFile::Base] object the file will be added to
```

Comment:
Could you add a comment explaining the purpose of the service?  Something like:
```
This Service conforms to the signature of `Hydra::Derivatives::PersistOutputFileService`.  
The purpose of this Service is for use an alternative to the default Hydra::Derivatives::PersistOutputFileService.  It's necessary because the default behavior in Hydra::Derivatives assumes that you're using LDP Basic Containment.  Hydra::Works::GenericFiles use IndirectContainment.  This Service handles that case.
```